src/app/containers/MediaPlayer/helpers/logEmbedSourceStatus/index.js

```diff
@@ -0,0 +1,20 @@
+import nodeLogger from '#lib/logger.node';
+import { MEDIA_PLAYER_RESPONSE } from '#lib/logger.const';
+
+const logger = nodeLogger(__filename);
+
+async function logEmbedSourceStatus(embedSource) {
```

Comment:
Can we use fat arrow notation here, for consistency?

```
const logEmbedSourceStatus = async embedSource => {
```

Reply:
Can we also update the name of the parameter to be `url`, so that you can just set it on the `data` object directly e.g.
```
const data = {url, status};
```

Reply:
Actually, could we also include the assetUrl and assetType - you should be able to pass this in from the MediaPlayer.  This way we can link the embedURL to the respective CPS asset. 

See https://github.com/bbc/simorgh/pull/6332 for further info.

Reply:
Done. I'm assuming the assetUrl is just the embedSource. I can't seem to explicitly find an assetUrl in the MediaPlayer.